pkgs/applications/science/machine-learning/starspace/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, stdenv, fetchFromGitHub, boost165}:
+
+stdenv.mkDerivation rec {
+  pname = "starspace";
+  version = "20210117-${lib.strings.substring 0 7 rev}";
+  rev = "8aee0a950aa607c023e5c91cff518bec335b5df5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "starspace";
+    inherit rev;
+    sha256 = "0sc7a37z1skb9377a1qs8ggwrkz0nmpybx7sms38xj05b702kbvj";
+  };
+
+  nativeBuildInputs = [ boost165 ];
```

Comment:
This seems like to belong rather to `buildInputs`.

Also are you use newer boost cannot be used? If yes, please provide a comment describing the issue when building with newer boost (i.e. `boost` package).

Reply:
> This seems like to belong rather to buildInputs.
Applied in f8b69fa6d2fcf54548795e3e04845f304efb3a30

> Also are you use newer boost cannot be used?
[The documentation for StarSpace explicitly mentions Boost 1.63](https://github.com/facebookresearch/StarSpace#requirements), which was deleted from nixpkgs some time ago. So I used 1.65 which seems to work. 
No idea if if works with a newer version but why would we want to deviate so much from the official requirements?